composer.json

```
@@ -4,15 +4,15 @@
     "type": "cakephp-plugin",
     "license": ["MIT"],
     "require": {
-        "php": ">=5.6",
+        "php": ">=7.2",
         "ext-pdo": "*",
         "ext-json": "*",
-        "cakephp/cakephp": "^3.4",
-        "psr/log": "^1.0"
+        "psr/log": "^1.0",
+        "cakephp/cakephp": "4.0.*"
```

Please change to like bellow, include CakePHP 4.1.

```suggestion
        "cakephp/cakephp": "^4.0"
```